### PR TITLE
WinUI 890011 - Updated API links for Scheduler tooltip content-hotfix

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
I have updated the API links for the Scheduler tooltip.

### CODE REVIEW

**Comment by TamilarasanGunasekaran** [comment]
Hi @Karthickmani97 ,
These changes are already been reviewed.
PR link - https://github.com/syncfusion-content/winui-docs/pull/844

File: winui/Scheduler/Appointments.md [hunk 1]
```diff
@@ -762,29 +762,32 @@ An interactive tooltip provides additional details about the appointments on hov
 
 ### Enable tooltip for appointments
 
-To enable tooltip for the scheduler appointments, use the `EnableTooltip` property of `SfScheduler.` By default, `EnableTooltip` is set to `false.` To provide users with additional information or context about appointments, simply set this property to true.
+To enable tooltip for the scheduler appointments, use the [EnableToolTip](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html#Syncfusion_UI_Xaml_Scheduler_SfScheduler_EnableToolTip) property of [SfScheduler](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html). By default, [EnableToolTip](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html#Syncfusion_UI_Xaml_Scheduler_SfScheduler_EnableToolTip) is set to `false.` To provide users with additional information or context about appointments, simply set this property to true.
 
 {% tabs %}
-{% highlight xaml %}
+{% highlight xaml tabtitle="MainWindow.xaml" %}
```
Comment (Karthickmani97) [inline winui/Scheduler/Appointments.md:768]: Remove `tabtitle` in all places.

File: winui/Scheduler/Day-Week-Views.md [hunk 2]
```diff
@@ -499,7 +499,7 @@ Customize the default appearance of view header by setting the [ViewHeaderTempla
 
 ## Cell right padding
 
-You can customize the spacing between an appointment and the right border of its cell to enhance interaction with a scheduler that contains appointments.
+To enhance interaction with a scheduler's appointments, you can customize the spacing between appointments and the right border of their cells using the [CellRightPadding](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.DaysViewSettings.html#Syncfusion_UI_Xaml_Scheduler_DaysViewSettings_CellRightPadding) property of [DayViewSettings](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.DaysViewSettings.html) in the scheduler.
```
Comment (Karthickmani97) [inline winui/Scheduler/Day-Week-Views.md:502]: Rename scheduler - [SfScheduler](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html)

File: winui/Scheduler/Month-View.md [hunk 3]
```diff
@@ -596,7 +596,7 @@ You can customize the default appearance of more appointments indicator in a mon
 
 ## Cell right padding
 
-You can customize the spacing between an appointment and the right border of its cell to enhance interaction with a scheduler that contains appointments.
+To enhance interaction with a scheduler's appointments, you can customize the spacing between appointments and the right border of their cells using the [CellRightPadding](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.MonthViewSettings.html#Syncfusion_UI_Xaml_Scheduler_MonthViewSettings_CellRightPadding) property of [MonthViewSettings](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.MonthViewSettings.html) in the scheduler.
```
Comment (Karthickmani97) [inline winui/Scheduler/Month-View.md:599]: Rename scheduler - [SfScheduler](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html)

File: winui/Scheduler/Timeline-Views.md [hunk 4]
```diff
@@ -503,7 +503,7 @@ this.Schedule.TimelineViewSettings.TimelineAppointmentHeight = 100;
 
 ## Cell bottom padding
 
-You can customize the spacing between an appointment and the bottom border of its cell to enhance interaction with a scheduler that contains appointments.
+To enhance interaction with a scheduler's appointments, you can customize the spacing between appointments and the bottom border of their cells using the [CellBottomPadding](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.TimelineViewSettings.html#Syncfusion_UI_Xaml_Scheduler_TimelineViewSettings_CellBottomPadding) property of [TimelineViewSettings](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.TimelineViewSettings.html) in the scheduler.
```
Comment (Karthickmani97) [inline winui/Scheduler/Timeline-Views.md:506]: Rename scheduler - [SfScheduler](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html)

Comment (TamilarasanGunasekaran) [reply winui/Scheduler/Timeline-Views.md]: Updated.

File: winui/Scheduler/Month-View.md (hunk 3 above)
Comment (TamilarasanGunasekaran) [reply winui/Scheduler/Month-View.md]: Updated.

File: winui/Scheduler/Day-Week-Views.md (hunk 2 above)
Comment (TamilarasanGunasekaran) [reply winui/Scheduler/Day-Week-Views.md]: Updated

File: winui/Scheduler/Day-Week-Views.md [hunk 5]
```diff
@@ -499,7 +499,7 @@ Customize the default appearance of view header by setting the [ViewHeaderTempla
 
 ## Cell right padding
 
-You can customize the spacing between an appointment and the right border of its cell to enhance interaction with a scheduler that contains appointments.
+To enhance interaction with a scheduler's appointments, you can customize the spacing between appointments and the right border of their cells using the [CellRightPadding](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.DaysViewSettings.html#Syncfusion_UI_Xaml_Scheduler_DaysViewSettings_CellRightPadding) property of [DayViewSettings](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.DaysViewSettings.html) in the [SfScheduler](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html).
```
Comment (Karthickmani97) [inline winui/Scheduler/Day-Week-Views.md:502]: Update SfScheduler WinUI API link - remove WPF

Comment (TamilarasanGunasekaran) [reply winui/Scheduler/Day-Week-Views.md]: Updated.

**Review by Karthickmani97 — APPROVED** [review]
This is fine.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**